pkg/cmd/compose/compose.go

```diff
@@ -125,7 +125,7 @@ func New(client *containerd.Client, globalOptions types.GlobalCommandOptions, op
 				ipfsPath = dir
 			}
 			_, err = ipfs.EnsureImage(ctx, client, stdout, stderr, globalOptions.Snapshotter, scheme, ref,
-				pullMode, ocispecPlatforms, nil, quiet, ipfsPath)
+				pullMode, ocispecPlatforms, nil, quiet, ipfsPath, &imgutil.RemoteSnapshotterFlags{})
```

Comment:
should we also put a `RemoteSnapshotterFlags` in `ImagePullOptions`, and make `SociIndexDigest` part of `RemoteSnapshotterFlags`?